src/draw/sw/lv_draw_sw.c

```diff
@@ -313,6 +315,11 @@ static int32_t dispatch(lv_draw_unit_t * draw_unit, lv_layer_t * layer)
         return -1;
     }
 
+    extern int flush_cnt;
+    if(flush_cnt > 0) {
+        LV_LOG_WARN("!!!!!!!!!!!!!!!!!!!!!!!!!!!!! flush_cnt: %d", flush_cnt);
```

Comment:
Funny, and very important warning message!!!!!!!!!!!!!!!  :-)
I think we can remove this "debugging" code part.

Reply:
Ah, this is a test code, used to check whether the GPU drawing is completed before software rendering. I found that there are still problems, so I will convert it to a draft~